_posts/2023-11-08-securing-wildfly-apps-oidc-openshift.adoc

```diff
@@ -0,0 +1,9 @@
+---
+layout: post
+title: 'Securing WildFly Apps with OIDC on OpenShift'
+date: 2023-11-08
```

Comment:
Please change the date here and the file name to the date you update this. When you push please ping me in Zulip.